seatunnel-engine/seatunnel-engine-common/src/main/java/org/apache/seatunnel/engine/common/Constant.java

```diff
@@ -22,5 +22,17 @@ public class Constant {
 
     public static final String SEATUNNEL_ID_GENERATOR_NAME = "SeaTunnelIdGenerator";
 
-    public static final String DEFAULT_SEATUNNEL_CLUSTER_NAME = "SeaTunnel";
+    public static final String DEFAULT_SEATUNNEL_CLUSTER_NAME = "seatunnel";
+
+    /**
+     * The default port number for the cluster auto-discovery mechanism's
+     * multicast communication.
+     */
+    public static final int DEFAULT_SEATUNNEL_MULTICAST_PORT = 53326;
```

Comment:
Is this parameter configurable? Often users may use a specific port.

Reply:
> Is this parameter configurable? Often users may use a specific port.

Yes, the port can be overwrite by this code

```
    public SeaTunnelConfig build(SeaTunnelConfig config) {
        try {
            parseAndBuildConfig(config);
        } catch (Exception e) {
            throw ExceptionUtil.rethrow(e);
        }
        config.setHazelcastConfig(ConfigProvider.locateAndGetMemberConfig(getProperties()));
        return config;
    }
```